spec/lib/topics_filter_spec.rb

```diff
@@ -1500,6 +1506,38 @@
           end
         end
       end
+
+      context "for DiscoursePluginRegistry.custom_filter_mappings" do
+        describe "when extending order:{col}" do
+          fab!(:earlier_topic) { Fabricate(:topic, bumped_at: 2.hours.ago) }
+          fab!(:now_topic) { Fabricate(:topic, bumped_at: Time.now) }
+
+          before do
+            plugin.add_filter_custom_filter(
+              "order:bumped",
+              &->(scope, is_asc: false) { scope.order("bumped_at #{is_asc ? "ASC" : "DESC"}") }
```

Comment:
I think this is OK but how can we communicate that in the API's documentation? I think this is important because the arguments passed to the block is different based on what the filter is. 

https://github.com/discourse/discourse/blob/66a23180d4655c0d0312e0a6485b5dbad291952a/lib/topics_filter.rb#L88

Reply:
Yea that might be better.